src/Compilers/VisualBasic/Portable/Binding/Binder.vb

```diff
@@ -452,6 +453,60 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return type.GetUseSiteErrorInfo()
         End Function
 
+        Private Function GetInternalXmlHelperType(syntax As VisualBasicSyntaxNode, diagBag As DiagnosticBag) As NamedTypeSymbol
+            Debug.Assert(Me.ContainingType IsNot Nothing)
+            Dim containingAssembly = Me.ContainingType.ContainingAssembly
```

Comment:
Should the lookup be performed within the module rather than within the entire assembly?